store/src/java-test/com/zimbra/cs/imap/ImapLoadBalancingMechanismTest.java

```diff
@@ -84,6 +84,9 @@ public void ClientIpHashMechanismIpHashFromPool()
         Assert.assertEquals(s1, mech.getImapServerFromPool(req, pool));
         headers.put(ImapLoadBalancingMechanism.ClientIpHashMechanism.CLIENT_IP, "127.0.0.4");
         Assert.assertEquals(s2, mech.getImapServerFromPool(req, pool));
+        headers.put(ImapLoadBalancingMechanism.ClientIpHashMechanism.CLIENT_IP, "192.168.56.1");
+        req = new MockHttpServletRequest(null, null, null, 123, "127.0.0.1", headers);
+        Assert.assertEquals(s1, mech.getImapServerFromPool(req, pool));
```

Comment:
I have updated the unit test to use the form with a description of what is failing.